engine/random.cpp

```diff
@@ -0,0 +1,42 @@
+#include "random.h"
+
+namespace pong
+{
+
+static std::random_device rd;
+static std::mt19937 generator(rd());
```

Comment:
I would recommend adding this as a member of the class. Static can get messy with multithreading, I think it probably makes sense to have it be passed around in the obj itself